xcell/mappers/mapper_BOSSLOWZ.py

```diff
@@ -0,0 +1,37 @@
+from .mapper_base import MapperSDSS
+import healpy as hp
+
+
+class MapperBOSSLOWZ(MapperSDSS):
```

Comment:
If the treatment of CMASS and LOWZ is **exactly** the same, it might make sense to just have a single `MapperBOSS` class and move on. Let's make sure that this is the case (i.e. let's look at the papers and see if there are subtleties) before doing so.

Reply:
Done